Learn/utilities/Switcher.swift

```diff
@@ -22,8 +22,8 @@ class Switcher {
         appDelegate.window?.rootViewController = rootVC
     }
     
-    static func loggedIn() -> Bool {
-        guard let token =  UserDefaults.standard.string(forKey: "token") else { return false }
+    static func isLoggedIn() -> Bool {
+        guard UserDefaults.standard.string(forKey: "token") != nil else { return false }
```

Comment:
This solved a notification warning about how `let token` was never used. Not sure if this is a good pattern in Swift, but figured it was better than the warning.